src/native_main.nim

```diff
@@ -152,15 +182,48 @@ proc handleMessage(msg: MessageRecv): string =
                 reply.code = some(2)
 
         of "move":
-            let
-                dest = expandTilde(msg.to.get())
-                src  = expandTilde(msg.`from`.get())
-            if fileExists(dest):
+            let src = expandTilde(msg.from.get())
+            let dst = expandTilde(msg.to.get())
+
+            if fileExists(dst):
                 reply.code = some(1)
             else:
                 try:
-                    moveFile(src, dest)
-                    reply.code = some(0)
+                    when defined(macosx):
+                        if DEBUG:
+                            write(stderr, ">> MacOS detected ...\n")
+                        var removeAttrErr = 0
+                        let attrsToRemove = @[
+                            "com.apple.quarantine",
+                            "com.apple.metadata:kMDItemWhereFroms"
+                        ]
+                        for attr in attrsToRemove:
+                            removeAttrErr = removeFileAttribute(src, attr)
+                            if removeAttrErr != 0:
+                                break
+
+                        if removeAttrErr == 0:
+                            var mvErr = 0
+                            var mvCmd = ""
+                            if DEBUG:
+                                mvCmd = quoteShellCommand(["mv", "-v", "-f", src, dst])
+                                write(stderr, ">> mvCmd == " & $mvCmd & "\n")
+                            else:
+                                mvCmd = quoteShellCommand(["mv", "-f", src, dst])
+
+                            # moveFile(src, dst)
+                            mvErr = execCmd(mvCmd)
```

Comment:
I'm wondering if we should put the flags back here. If people didn't want the flags presumably they'd disable them at a system level? It seems like it shouldn't be a decision that Tridactyl makes